HB/export.elpi

```diff
@@ -96,8 +97,11 @@ compute-filter (some S) MFilter :- % S is a component of the current modpath
 compute-filter (some S) MFilter :-
   coq.locate-module S M,
   coq.modpath->path M MFilter.
+
+% @gares: same predicate in synterp: code clone?
+func compute-filter.aux list string, list string -> list string.
 compute-filter.aux [S|_] [S] [S] :- !.
```

Comment:
Is it intentional that this predicate is also declared identically in utils-synter.elpi?